### PR TITLE
feat(import): add import endpoints

### PR DESCRIPTION
**Pull request by peterdeme**

### CODE REVIEW

File: stream_chat/client.py [hunk 1]
```diff
@@ -598,3 +605,17 @@ def delete_push_provider(self, provider_type: str, name: str) -> StreamResponse:
 
     def list_push_providers(self) -> StreamResponse:
         return self.get("push_providers")
+
+    def create_import_url(self, filename: str) -> StreamResponse:
+        return self.post("import_urls", data={"filename": filename})
```
Comment (gumuz) [inline stream_chat/client.py:610]: maybe it makes sense to wrap this into a helper that does the uploading too, what do you think @ferhatelmas ?

Comment (peterdeme) [reply stream_chat/client.py:610]: we dont have it in other (js, go) sdk's. I think it's better to leave it to the customer

File: stream_chat/async_chat/client.py [hunk 2]
```diff
@@ -633,6 +639,20 @@ async def delete_push_provider(
     async def list_push_providers(self) -> StreamResponse:
         return await self.get("push_providers")
 
+    async def create_import_url(self, filename: str) -> StreamResponse:
+        return await self.post("import_urls", data={"filename": filename})
+
+    async def create_import(
+        self, filepath: str, mode: Literal["insert", "upsert"] = "upsert"
+    ) -> StreamResponse:
+        return await self.post("imports", data={"path": filepath, "mode": mode})
```
[the remark this answers is not in the document]
Comment (peterdeme) [reply stream_chat/async_chat/client.py:648]: ✅

**Review by gumuz — APPROVED** [review]

File: stream_chat/client.py (hunk 1 above)
Comment (ferhatelmas) [reply stream_chat/client.py:610]: consistent might be better

Comment (gumuz) [reply stream_chat/client.py:610]: Is also saw that Peter included an example in the doc string. LGTM 🙌

**Pull request merged**